browser.js

```diff
@@ -107,6 +107,28 @@ define(function(require, exports, module) {
             
             iframe.src = url;
         }
+
+        function setTitle(session, tab, path) {
+            // remember default title between reloads
+            if (!session.doc.title) {
+                session.doc.title = "[B] " + path;
+                session.defaultTitle = true;
+            }
+            // update title if default is used
+            else if (session.defaultTitle) {
+                tab.title = "[B] " + path;
+            }
+
+            // remember default tooltip between reloads
```

Comment:
this whole section can be removed, tooltip falls back to title anyway https://github.com/c9/core/blob/41328459397e9c4ec214db3a5fabad99a3e23601/plugins/c9.ide.editors/pane.js#L248